src/metabase/automagic_dashboards/core.clj

```diff
@@ -1106,22 +1107,22 @@
   humanize-filter-value (fn [_ [op & args]]
                           (qp.util/normalize-token op)))
 
-(def ^:private unit-name (comp {:minute-of-hour  "minute"
-                                :hour-of-day     "hour"
-                                :day-of-week     "day of week"
-                                :day-of-month    "day of month"
-                                :day-of-year     "day of year"
-                                :week-of-year    "week"
-                                :month-of-year   "month"
-                                :quarter-of-year "quarter"}
+(def ^:private unit-name (comp {:minute-of-hour  (tru "minute")
+                                :hour-of-day     (tru "hour")
+                                :day-of-week     (tru "day of week")
+                                :day-of-month    (tru "day of month")
+                                :day-of-year     (tru "day of year")
+                                :week-of-year    (tru "week")
+                                :month-of-year   (tru "month")
+                                :quarter-of-year (tru "quarter")}
                                qp.util/normalize-token))
 
 (defn- field-name
   ([root field-reference]
    (->> field-reference (field-reference->field root) field-name))
   ([{:keys [display_name unit] :as field}]
    (cond->> display_name
-     (and (filters/periodic-datetime? field) unit) (format "%s of %s" (unit-name unit)))))
+     (and (filters/periodic-datetime? field) unit) (tru "{0} of {1}" (unit-name unit)))))
```

Comment:
Looks like there is supposed to be two parameters  here but I only see one.

Reply:
The second one is provided by cond->> (perhaps not the clearest idiom, granted)